core/src/main/java/org/owasp/dependencycheck/reporting/ReportGenerator.java

```diff
@@ -258,6 +258,7 @@ private VelocityContext createContext(String applicationName, List<Dependency> d
         ctxt.put("scanDateJunit", scanDateJunit);
         ctxt.put("enc", new EscapeTool());
         ctxt.put("rpt", new ReportTool());
+        ctxt.put("hash", new HashTool());
```

Comment:
Use existing objects:

```suggestion
        ctxt.put("hash", org.owasp.dependencycheck.utils.Checksum.class);
```